packages/logs/src/boot/logsPublicApi.ts

```diff
@@ -55,7 +55,7 @@ export function makeLogsPublicApi(startLogsImpl: StartLogs) {
   let handleLogStrategy: StartLogsResult['handleLog'] = (
     logsMessage: LogsMessage,
     logger: Logger,
-    savedCommonContext = deepClone(buildCommonContext()),
+    savedCommonContext = getCommonContext(),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏`deepClone` not needed anymore?

Reply:
It wasn't needed: [`buildCommonContext`](https://github.com/DataDog/browser-sdk/blob/37fd6d92c0f2836bff9fa983a95a3a4e5e7ac71d/packages/logs/src/domain/contexts/commonContext.ts) is built from scratch + uses `contextManager.getContext()`, which already uses `deepClone`. RUM already works similarly.